.github/workflows/workflow.yml

```
@@ -20,14 +21,32 @@ jobs:
         with:
           submodules: 'recursive'
 
-      - name: Install LLVM repositories
+      - name: Install LLVM repositories Ubuntu
+        if: runner.os == 'Linux'
         run: sudo add-apt-repository -y 'deb http://apt.llvm.org/focal/ llvm-toolchain-focal-12 main'
 
-      - name: Install required OS packages
+      - name: Install required Ubuntu packages
+        if: runner.os == 'Linux'
         run: sudo apt-get update && sudo apt-get install -yq build-essential pkg-config zlib1g-dev libssl-dev libboost-system-dev libboost-filesystem-dev libboost-program-options-dev libboost-system-dev libboost-test-dev libjsoncpp-dev libsecp256k1-dev clang-12
 
+      - name: Install required MacOS packages
+        if: runner.os == 'macOS'
+        run: brew update && brew upgrade && brew install llvm cmake openssl pkg-config boost autoconf automake libtool jsoncpp domt4/crypto/secp256k1
+
       - name: ScillaRTL Testsuite Release Build
```

```suggestion
      - name: ScillaRTL Testsuite Release Build [ Ubuntu ]
```

And similarly others, so that when we look at the Actions page, we know what is being built.